targets/targets.json

```diff
@@ -4483,6 +4827,10 @@
             "LWIP"
         ],
         "device_name": "MIMXRT1052",
+        "mbed_ram_start": "0x20000000",
```

Comment:
Does this device really not have any ROM/flash or was this just left out of the `index.json` on accident? Please check the datasheet. Maybe `IRAM2` in `index.json` is actually supposed to be `IROM1`? This could be another `index.json` bug.